packages/react-ui-components/src/stories/Components/TreeItem.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import { EIconName, EOtherIconName } from '@kelvininc/ui-components';
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvTreeItem, ETreeItemStates } from '../../components';
+
+// Required to have the correct TagName in the code sample
+KvTreeItem.displayName = 'KvTreeItem';
+
+export default {
+	title: 'Navigation/Tree/Tree Item',
+	component: 'kv-tree-item',
+	argTypes: {
+		icon: {
+			control: 'select',
+			options: [...Object.values(EIconName), ...Object.values(EOtherIconName)]
+		},
+		iconState: {
+			control: 'select',
+			options: Object.values(ETreeItemStates)
+		},
+		counterState: {
+			control: 'select',
+			options: Object.values(ETreeItemStates)
+		}
+	},
+	parameters: {
+		notes: require('@ui-notes/tree-item/readme.md')
+	}
+};
+
+const handleToggleExpand = (e: any) => console.log('onToggleExpand', e);
```

Comment:
We should move the actions callbacks to `argTypes` with this options:

```
onItemClick: {
		action: 'clicked'
},
onToggleExpand: {
		action: 'clicked'
}
```

After that we can remove from the template the bind for this function.